users/schicklr/index.md

```diff
@@ -5,8 +5,39 @@ author: Robert Schickler
 ---
 Robert Schickler
 ================================
-I like pizza.
+<img src="images/schickler.jpg">
```

Comment:
If you want the exact dimensions your picture has, you can omit `height="X" width="Y"`. Just consider how big it'll be on the page. For instance, my picture dimensions are 500x500